### PR TITLE
Add update mutations

### PR DESCRIPTION
**Pull request by Nyakaru**
- add school update mutation
- add user update mutation
- add super user role

### CODE REVIEW

File: src/resolvers/mutations/school.js [hunk 1]
```diff
@@ -49,6 +49,24 @@ const school = async (_, { input }, context, _info) => {
...
 
+/**
+ * @param {any} _
+ * @param {any} input
+ * @param {import('../../..').IRequestContext} context
+ * @param {any} _info
+ */
+const updateSchool = async (_, input, context, _info) => {
+    try {
+        const school = await context.prisma.updateSchool({ where: input['where'], data: input['data'] }).$fragment(utils.schoolFragment);
```
Comment (MbugwaSami) [inline src/resolvers/mutations/school.js:60]: what happens if the school does not exist?

Comment (Nyakaru) [reply src/resolvers/mutations/school.js:60]: Throw a graphql error

Comment (MbugwaSami) [reply src/resolvers/mutations/school.js:60]: will we be able to handle this on UI? I see a scenario where a user updates something deleted by another user.

Comment (Nyakaru) [reply src/resolvers/mutations/school.js:60]: Yes plus this mutation sisi ndio tutatumia and we will be choosing from a dropdown of existing schools

**Pull request merged**